editors/code/src/highlighting.ts

```diff
@@ -45,13 +41,10 @@ export function activateHighlighting(ctx: Ctx) {
             const client = ctx.client;
             if (!client) return;
```

Comment:
We should get rid of these if statements, since the client is not nullable today...
But this deserves a separate PR.

Reply:
yup, there's FIXME about this in main.ts